phpstan-baseline-standard.neon

```diff
@@ -3710,6 +3615,11 @@ parameters:
 			count: 1
 			path: library/Businessprocess/Storage/Storage.php
 
+		-
+			message: "#^Property Icinga\\\\Module\\\\Businessprocess\\\\Storage\\\\Storage\\:\\:\\$config with generic class Icinga\\\\Data\\\\ConfigObject does not specify its types\\: TValue$#"
```

Comment:
This is related to a recent change in Icinga Web and should be properly fixed.

Reply:
Fixed in https://github.com/Icinga/icingaweb2-module-businessprocess/pull/446